docs/source/AdministratorGuide/CommandReference/dirac-admin-update-instance.rst

```diff
@@ -0,0 +1,18 @@
+.. _admin_dirac-admin-update-instance:
```

Comment:
I thought all that will be generated automatically ? @andresailer

Reply:
yes, it should be created automatically.
But the docstring should be expanded to describe what this does.

Reply:
https://github.com/DIRACGrid/DIRAC/pull/4800/checks?check_run_id=1349732434#step:5:241

Reply:
but the addition to the index is needed?

Reply:
Yes, the admin index cannot be created automatically in any sensible way.